packages/agoric-cli/test/inter-cli.test.js

```diff
@@ -2,11 +2,11 @@
 /* eslint-env node */
 import '@endo/init';
```

Comment:
Have you tried [@endo/init/legacy.js](https://unpkg.com/browse/@endo/init@1.1.3/legacy.js)? I'd expect axios to work with that, because `@agoric/rpc` itself already uses axios. It would still be great to get your patch into axios itself

Reply:
Thank you! That will help so much with #9200.